botogram/bot.py

```diff
@@ -144,6 +148,16 @@ def message_edited(self, func):
         self._main_component.add_message_edited_hook(func)
         return func
 
+    def channel_post(self, func):
+        """Add a channel post hook"""
+        self._main_component.add_channel_post_hook(func)
+        return func
+
+    def edited_channel_post(self, func):
```

Comment:
Maybe it's better to call this `channel_post_edited`, for the sake of consistency with `message_edited`.